.gitignore

```diff
@@ -2,3 +2,5 @@ promtool
 yq
 test/promql/extracted-rules.yaml
 pint
+.venv
+Pipfile.lock
```

Comment:
This must NOT be in gitignore. The idea is that we're locking the dependencies and then all envs sync according to the locked dependencies.

Reply:
> This must NOT be in gitignore. The idea is that we're locking the dependencies and then all envs sync according to the locked dependencies.

Thank you for your explanation. It has been removed from gitgnore and the Pipfile.lock has been updated with all the required versions.